modules/runner/src/main/java/org/apache/ignite/internal/configuration/storage/LocalFileConfigurationStorage.java

```diff
@@ -108,10 +131,24 @@ public LocalFileConfigurationStorage(Path configPath, ConfigurationTreeGenerator
         this.configPath = configPath;
         this.generator = generator;
         this.tempConfigPath = configPath.resolveSibling(configPath.getFileName() + ".tmp");
+        this.module = null;
```

Comment:
All this code can be replaced with `this(configPath, generator, null)`

Reply:
The second constructor was removed